src/collection.js

```diff
@@ -9,12 +9,33 @@ var read = Promise.denodeify(fs.readFile)
 
 module.exports = function(options) {
   var root = options.path
+  var user = options.user || '_default'
+  var bareroot = path.join(root,'_bare')
   var data = {}
-  var repo = new Repository(root)
+  var barerepo = new Repository(bareroot)
+  var userroot = path.join(root, user)
+
+  var repo = new Repository(userroot)
+
+  // Directory setup for empty repository
+  fs.lstat(bareroot, function(lstatErr, stats) {
+    if (!stats){
+      fs.mkdirSync(root)
```

Comment:
also think this should happen on a usage, don't want to be doing anything until save is called really